net-misc/ipv6calc/files/ipv6calc-4.3.2-ldconfig_musl.patch

```diff
@@ -0,0 +1,27 @@
+https://github.com/pbiering/ipv6calc/pull/49
+diff --git a/databases/lib/Makefile.in b/databases/lib/Makefile.in
+index ed3c55d..a57a080 100644
+--- a/databases/lib/Makefile.in
++++ b/databases/lib/Makefile.in
+@@ -91,7 +91,7 @@ ifeq ($(SHARED_LIBRARY), yes)
+ 		echo "Install shared library"
+ 		mkdir -p $(DESTDIR)$(libdir)
+ 		install -m 755 libipv6calc_db_wrapper.so.@PACKAGE_VERSION@ $(DESTDIR)$(libdir)
+-		/sbin/ldconfig -n $(DESTDIR)$(libdir)
++		/sbin/ldconfig $(DESTDIR)$(libdir)
```

Comment:
Better to drop it entirely for our purposes (that part won't be upstreamable, I guess), as ldconfig may not even exist there on prefix..

Reply:
Right. Not needed in fact after trying.